### PR TITLE
Soupster89 extended techniques mod

### PR DESCRIPTION
**Pull request by Soupster89**
#### Summary
SUMMARY: Mods "Adds various new techniques to spice up melee combat"```  

#### Purpose of change
Melee combat is rather one track as most weapons do not have techniques, these techniques can be used as building blocks to make many weapons have new attacks which can lead to more interesting play styles.

#### Describe the solution
 Eventually, the mod will have the tools and melee .jsons filled out with the new techniques instead of just the techniques without anything attached to them.

### CODE REVIEW

File: data/mods/techniques.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        "id" : "HOOK",
+        "name" : "hOOK",
+        "mult_bonuses" : [
+            ["movecost", 0.85],
+            ["damage", "bash", 0.66],
+            ["damage", "cut", 0.76],
+            ["damage", "stab", 0.86]
+        ],
+        "flat_bonuses": [
+            ["movecost", "str", -0.25"],
```
Comment (anothersimulacrum) [inline data/mods/techniques.json]: You've got a single " that need to be removed after the -0.25 here

File: data/mods/techniques.json [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        "melee_allowed" : true,
+        "knockback_dist": 1,
+        "mult_bonuses" : [
+            ["movecost", 0.70],
+            ["damage", "bash", 0.55],
+            ["damage", "cut", 0.55],
+            ["damage", "stab", 0.55]
+        ],
+        "flat_bonuses": [
+            ["movecost", "str", -0.25"],
```
Comment (anothersimulacrum) [inline data/mods/techniques.json]: As well as here, lines 36-39

File: data/mods/techniques.json [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        "melee_allowed" : true,
+        "crit_tec": true,
+        "mult_bonuses" : [
+            ["movecost", 1.15],
+            ["damage", "bash", 1.05],
+            ["damage", "cut", 1.25],
+            ["damage", "stab", 1.05]
+        ],
+        "flat_bonuses": [
+            ["movecost", "dex", -0.20"],
```
Comment (anothersimulacrum) [inline data/mods/techniques.json]: And here, as well as lines 89, 112, 135-6.

File: modinfo.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[
```
Comment (anothersimulacrum) [inline modinfo.json]: This is a duplicate modinfo.json file that is in the main directory.

File: techniques.json [hunk 5]
```diff
@@ -0,0 +1,162 @@
+[
```
Comment (anothersimulacrum) [inline techniques.json]: This is a duplicate techniques.json file that is in the main directory.

Comment (Soupster89) [reply techniques.json]: Thanks for the help, first time doing this. I'm not entirely sure if I deleted the correct files, doI jusyt do in and trash the duplicates or start a new branch?

Comment (anothersimulacrum) [reply techniques.json]: You deleted the correct files, and did it right, but the issues in the first three review comments still are not resolved.

**Pull request closed without merging**